### DISCUSSION TITLE
Switch on the Display via MQTT message

### DISCUSSION THREAD

**Discussion by Sprocket02** [seaky/nspanel_pro_tools_apk#77] [Q&A]
Is it also possible to wake up the display via an MQTT command with the new version? I would like to use a presence detector to switch on the display when I enter the room.

**Comment by gajda** [comment]
I would also really like this featufe.

**Reply by seaky** [reply]
Yes, v2.2 will support incoming commands as well. 

Check documentation "Home Assistant integration"

Commands already in v2.2 are
- reboot
- wake up
- sleep
- play audio

**Comment by Sprocket02** [comment]
That sounds very good! Is it possible to become a beta tester for this feature?
